EIPS/eip-00.md

```diff
@@ -0,0 +1,790 @@
+---
+eip: 0
+title: Native Account Abstraction
+description: An account abstraction proposal that introduces consensus-layer protocol changes, instead of relying on higher-layer infrastructure.
+author: Vitalik Buterin (@vbuterin), Yoav Weiss (@yoavw), Alex Forshtat (@forshtat), Dror Tirosh (@drortirosh), Shahaf Nacson (@shahafn)
+discussions-to: https://www.google.com/
+status: Draft
+type: Standards Track
+category: Core
+created: 2023-09-01
+requires: 4337, 6780
+---
+
+## Abstract
+
+Combining the [EIP-2938](./eip-2938)
+and [ERC-4337](./eip-4337)
+into a comprehensive Native Account Abstraction proposal.
+
+We propose splitting the Ethereum transaction scope into multiple steps: validations, execution, and cleanup.
```

Comment:
"cleanup" (the equivalent of `postOp` in 4337) is a separate execution frame in this EIP, so I think it makes sense to mention it.  But I think it's a leftover from when we initially called the function `cleanup()`.  It has been renamed to `postPaymasterTransaction()` since, so maybe we should change it in the abstract as well.  @forshtat WDYT?